### ISSUE TITLE
Async file buffering

### ISSUE THREAD

**Issue by sm-Fifteen** [AlexWaveDiver/TranslaTale#15]
As it now stands, loading both string files when opening a project makes the UI unresponsive for some time, while the application is busy reading them and populating its string lists. Shouldn't we just create a list of proxy string objects and return immediately while another thread replaces them with the actual strings from the files (if that's possible in VB?), so the interface doesn't block while that's hapenning?

**Comment by AlexWaveDiver** [comment]
Yeah, that's on my ToDo list! In fact, we're already using non-locking threads in frmStringsConverter.vb
